src/main/java/com/mineinabyss/looty/ecs/systems/CooldownDisplaySystem.kt

```diff
@@ -29,7 +29,8 @@ object CooldownDisplaySystem : TickingSystem(interval = INTERVAL) {
         entity.parent?.with<Player>() { player ->
             player.sendActionBar(cooldownManager.incompleteCooldowns.entries.joinToString("\n") { (key, cooldown) ->
                 val length = Duration.milliseconds(cooldown.length)
-                val timeLeft = Duration.milliseconds((cooldown.endTime - System.currentTimeMillis()))
+                val timeLeft =
```

Comment:
Avoid changing indent where things were fine before, id revert this back to one line